core/store/src/lib.rs

```diff
@@ -151,6 +151,10 @@ impl Store {
     pub fn get_rocksdb(&self) -> Option<&RocksDB> {
         self.storage.as_rocksdb()
     }
+
+    pub fn get_rocksdb_statistics(&self) -> Option<String> {
```

Comment:
what is the format of the string?

Reply:
maybe rather than string, we should create out own struct? (even if it starts as a string wrapper at the beginning)

Reply:
+1

we actually do create our own sturuct (see `StatsValue` and `parse_statistics`), it’s just that it happens couple of layers above. 

It’s better to parse strings in structured data as early as possible instead.